src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetCultureCommand.cs

```diff
@@ -1,23 +1,75 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using System.Globalization;
 using System.Management.Automation;
 
 namespace Microsoft.PowerShell.Commands
 {
     /// <summary>
-    /// Returns the thread's current culture.
+    /// Returns:
+    ///     - the thread's current culture
+    ///     - culture by name
+    ///     - list of all supported cultures.
     /// </summary>
-    [Cmdlet(VerbsCommon.Get, "Culture", HelpUri = "https://go.microsoft.com/fwlink/?LinkID=113312")]
+    [Cmdlet(VerbsCommon.Get, "Culture", DefaultParameterSetName = DefaultParameterSet, HelpUri = "https://go.microsoft.com/fwlink/?LinkID=113312")]
     [OutputType(typeof(System.Globalization.CultureInfo))]
     public sealed class GetCultureCommand : PSCmdlet
     {
+        private const string DefaultParameterSet = "Default";
+        private const string NameParameterSet = "Name";
+        private const string ListAvailableParameterSet = "ListAvailable";
+
+        /// <summary>
+        /// Gets or sets culture names for which CultureInfo values are returned.
+        /// </summary>
+        [Parameter(ParameterSetName = NameParameterSet, Position = 0, ValueFromPipeline = true)]
+        [ValidateNotNull]
+        public string[] Name { get; set; }
+
+        /// <summary>
+        /// Gets or sets a switch to list all available cultures.
+        /// </summary>
+        [Parameter(ParameterSetName = ListAvailableParameterSet)]
+        public SwitchParameter ListAvailable { get; set; }
+
         /// <summary>
-        /// Output the current Culture info object.
+        /// Output:
+        ///     - the thread's current culture
+        ///     - culture by name
+        ///     - list of all supported cultures.
         /// </summary>
-        protected override void BeginProcessing()
+        protected override void ProcessRecord()
         {
-            WriteObject(Host.CurrentCulture);
+            switch (ParameterSetName)
+            {
+                case DefaultParameterSet:
+                    WriteObject(Host.CurrentCulture);
+
+                    break;
+                case NameParameterSet:
+                    try
+                    {
+                        foreach (var cultureName in Name)
+                        {
+                            CultureInfo ci = CultureInfo.GetCultureInfo(cultureName);
```

Comment:
This should use `new CultureInfo(cultureName, true)` to also reflect overrides for the current culture, for consistency with the other parameter sets. See https://github.com/PowerShell/PowerShell/pull/7702#issuecomment-420048195